### DISCUSSION TITLE
WebSocket is already in CLOSING or CLOSED state. 登入系统报错

### DISCUSSION THREAD

**Discussion by yuemo1** [illacloud/illa-builder#1017] [Q&A]
![image](https://user-images.githubusercontent.com/26081267/221452803-5b9ed231-eafd-4552-90e0-03179e2faab3.png)
登入系统老是报这个错，导致页面刷新，什么原因导致的

**Comment by yuemo1** [comment]
![image](https://user-images.githubusercontent.com/26081267/221453033-a1dd46e6-486d-4eba-81c9-88b0a3b919ba.png)
这个是日志

**Comment by owenlongbo** [comment]
Fixed.
